docs/odata-url-conventions/odata-url-conventions.md

```diff
@@ -1555,6 +1555,9 @@ The `eq`, `ne`, and `in` operators can be used with collection-valued
 operands, and the `eq` and `ne` operators can be used with operands of a
 structured type.
 
+The Boolean operators `and`, `or` and `not` MUST have Boolean operands,
+otherwise the expression is invalid.
```

Comment:
How are services supposed to treat invalid expressions?
- Treat as "not true" and continue, or
- Fail the request?

Reply:
See #1951